doc/lua_api.md

```diff
@@ -7815,6 +7821,10 @@ child will follow movement and rotation of that bone.
 
 ### Methods
 
+* `is_valid()`: returns whether the object is valid.
+   Objects may be invalidated either through explicit removal,
+   or implicitly between server steps by the engine.
```

Comment:
IIRC there's an "advice on object invalidation" section that should be updated alongside

it's also not mentioned that any objectrefs returned by engine functions are guaranteed to be valid at the time of returning
which then means that in most cases you won't need to deal with this

Reply:
thanks for pointing that out, edited the section

> it's also not mentioned that any objectrefs returned by engine functions are guaranteed to be valid at the time of returning

this seems to be already (somewhat implicitly) mentioned in said section so we're good?

Reply:
> this seems to be already (somewhat implicitly) mentioned in said section so we're good?

sure.
can you add a reference to that section too?